### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by HypnoMama**
# Description

UI work to more closely match the mockups.  Added separate down caret icon to dropdown.  Dropdown is responsive.  Handle text overflows.  Modify dropdown tests

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: **put the jira ticket # here**

### CODE REVIEW

File: maltd-frontend/src/components/page/MainPage/MainPage.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default function MainPage() {
   }
 
   function onButtonClick() {
-    fetch(`${baseUrl}/api/projects`)
+    fetch(`https://localhost:5001/api/projects`)
```
Comment (sdevalapurkar) [inline maltd-frontend/src/components/page/MainPage/MainPage.js]: this should not be changed, it should be `baseUrl`

File: api/README.md [hunk 2]
```diff
@@ -1,56 +1,58 @@
...
-               "Username":"",
-               "Password":""
-            },
-            {
-               "Type":"SharePoint",
-               "Resource":"",
-               "AuthorizationUri":"",
-               "ClientId":"",
-               "ClientSecret":"",
-               "Username":"",
```
Comment (sdevalapurkar) [inline api/README.md]: not sure if these changes should be in this pr, did you add them?

File: maltd-frontend/src/components/page/MainPage/MainPage.js [hunk 3]
```diff
@@ -117,7 +117,7 @@ export default function MainPage() {
           setDisabledButton(true);
           setDisabledInput(true);
 
-          fetch(`${baseUrl}/api/users/${value}`)
+          fetch(`https://localhost:5001/api/users/${value}`)
```
Comment (sdevalapurkar) [inline maltd-frontend/src/components/page/MainPage/MainPage.js]: no this needs to stay as `baseUrl`, cant be hard coded to `localhost` with a port

Comment (HypnoMama) [reply maltd-frontend/src/components/page/MainPage/MainPage.js]: Yeah, I accepted the wrong merge change!  Fixed and pushed.

Comment (HypnoMama) [reply maltd-frontend/src/components/page/MainPage/MainPage.js]: Thank you, fixed!

**Pull request merged**